docs/io/fits/usage/unfamiliar.rst

```diff
@@ -136,7 +137,7 @@ type specification (i.e. the value of the TFORM keyword) uses an extra letter
 
     rPt(max)
 
-where r is 0, 1, or absent, t is one of the letter code for regular table data
+where r is the column length (it can be absent), t is one of the letter code for regular table data
```

Comment:
I'm trying to add some information regarding the meaning of r values to the documentation as well  and I got a little confused about the possible values of r. The FITS Standard states that

"The repeat count r is the ASCII representation of a non-negative integer specifying the number of elements in field n. The default value of r is 1; the repeat count need not be present if it has the default value. A zero element count, indicating an empty field, is permitted. The data type T
specifies the data type of the contents of field n"

So, in general, we could have a case when r > 1 (such as the exemple code that creates a 2I column). Why in our case it may be only 0 or 1? Or why it is not applicable to variable length arrays?